slosh_test/run-tests.slosh

```diff
@@ -9,11 +9,11 @@
     (while (< i globals-len)
         (prn "Symbol: " globals.~i)
         (prn "===")
-        (prn "doc: " (doc globals.~i))
         (def doc (doc-map globals.~i))
-        (prn "Usage: " (str doc.:usage))
-        (prn "Description: " (str doc.:description))
-        (prn "Section: " (str doc.:section))
+        ;; for some reason printing usage for vec-slice causes rust to panic with 'Handle 2045 is not a string!'
+        ;; (prn "Usage: " (str doc.:usage))
```

Comment:
ran into an issue where if these were uncommented
`cargo test --workspace` would fail.

can you reproduce?

Reply:
That error most likely means something was garbage collected that should not have been.  Sometimes this is caused by doing lisp things from Rust (i.e. allocating something and not making it "sticky" can cause this if it is not rooted).  At first glace this looks like it is all on the Lisp side though so may be a legit bug.  I will have to dig into this to try to figure out where the GC breakdown is.